### DISCUSSION TITLE
Language Help

### DISCUSSION THREAD

**Discussion by pdns90** [deepgram/community#449] [General help] [answered]
Hi there, As we are giving interview in HINDI and we are getting transcription in Hindi but is there any way we can get transcription in English?

Thanks,
PDN

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @pdns90, our managed Whisper model provides some translation functionality. You can try submitting your request with `language=en`, and Whisper will transcribe your Hindi audio and return a translated transcript in English.
